rules-tests/DeadCode/Rector/Node/RemoveNonExistingVarAnnotationRector/Fixture/return_annotation.php.inc

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
please change file name to something like: `skip_direct_return_func_call.php.inc`

Reply:
I assume the other files are fine. I can't really tell a pattern with the non-skip files.

Reply:
Yes, using skip_ prefix is for consistency